commands/solenoid/SolenoidRetract.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc4904.standard.commands.solenoid;
+
+
+import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem;
+import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem.SolenoidState;
+
+public class SolenoidRetract extends SolenoidSet {
+	public SolenoidRetract(String name, SolenoidSubsystem system) {
+		super(name, system, SolenoidState.REVERSE);
```

Comment:
either change the name of the state to RETRACT or the name of the class and file to REVERSE (I prefer retract)